packages/protocol/contracts/governance/BlockchainParameters.sol

```diff
@@ -5,10 +5,12 @@ import "openzeppelin-solidity/contracts/ownership/Ownable.sol";
 import "../common/Initializable.sol";
 import "../common/UsingPrecompiles.sol";
 
+import "../../contracts-0.8/common/IsL2Check.sol";
+
 /**
  * @title Contract for storing blockchain parameters that can be set by governance.
```

Comment:
setBlockGasLimit and setIntrinsicGasForAlternativeFeeCurrency should also be deprecated

Reply:
I can add it, but it did not really seem that this was required, since these state variables are not referenced by the contract and these functions are only called by the owner.